src/main/java/edu/hm/hafner/analysis/ModuleDetector.java

```diff
@@ -181,6 +202,56 @@ private String parseBuildXml(final String buildXml) {
         return StringUtils.EMPTY;
     }
 
+    /**
+     * Returns the project name estimated from the build.gradle file path.
+     *
+     * @param buildScript
+     *         Gradle build.gradle file path
+     *
+     * @return the project name or an empty string if the name could not be resolved
+     */
+    private String parseGradle(final String buildScript) {
+        Path path = Paths.get(buildScript).normalize();
+        if (path.getNameCount() >= 2) {
+            Path parent = path.getParent();
```

Comment:
Maybe it would make more sense to use `getName(int)` to avoid SpotBugs finding a false positive (see https://ci.jenkins.io/job/Plugins/job/analysis-model/job/PR-362/6/)

Reply:
Sure, I'll use `path.getName(path.getNameCount() - 2)`, to refer to the parent.

Reply:
Looks like there were some more spotbugs warnings here, because of Paths.get(). I ended up switching to FilenameUtils, to resolve both issues.